crates/core/app/src/rpc.rs

```diff
@@ -109,6 +110,9 @@ pub fn router(
         .add_service(we(TransferQueryServer::new(ShieldedPoolServer::new(
             storage.clone(),
         ))))
+        .add_service(we(BankQueryServer::new(ShieldedPoolServer::new(
```

Comment:
it does feel weird to have a bank query service, since we do not have a cosmos bank module. this makes me worried that in the future an ecosystem user will attempt to make RPCs against them, expecting them to work like how the cosmos Bank module is specified.